polyply/src/simple_seq_parsers.py

```diff
@@ -253,6 +253,9 @@ def parse_ig(filepath):
         raise FileFormatError(msg)
 
     DNA, RNA, AA = _identify_residues(comments)
+    if set(clean_lines[0]).issubset(set(['A', 'C', 'G', 'T'])):
+        LOGGER.warning("Found only the letters A, C, G, T on first line. Are you missing the title line in your .ig file?")
+
     seq_graph = _parse_plain(clean_lines[1:], DNA=DNA, RNA=RNA, AA=AA)
```

Comment:
Dropping the first item in the list 'clean_lines' breaks the ig reader (tested it on the Single stranded circular DNA tutorial).

```suggestion
    seq_graph = _parse_plain(clean_lines, DNA=DNA, RNA=RNA, AA=AA)
```

Reply:
Just realized that in the ig file format, there should also be a sequence name between the comments and the sequence. So this is probably an error in the tutorial and not here?

Reply:
Yeah, I made this mistake a couple of times as well. The warning is literally issued a few lines before. Did it not print the warning? I also checked the online tutorial and to me it looks fine. From where did you copy the sequence?

Reply:
I copied the sequence from [here](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-Single-stranded-circular-DNA), where the comment does not specify that it is 'DNA'. 

I previously commented out the sequence name to troubleshoot and forgot I modified it. Adding 'DNA' to the comment line fixes the problems. Now I also realize that the warning a few lines above is actually for the sequence name... It makes sense now!

Reply:
@jan-stevens then it's resolved and ready to merge after your approval?

Reply:
Yes!